[2025-06-06T05:22:17.651+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LR_duration_prediction_training.train_model manual__2025-06-06T05:22:15.889534+00:00 [queued]>
[2025-06-06T05:22:17.659+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LR_duration_prediction_training.train_model manual__2025-06-06T05:22:15.889534+00:00 [queued]>
[2025-06-06T05:22:17.659+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-06-06T05:22:17.668+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): train_model> on 2025-06-06 05:22:15.889534+00:00
[2025-06-06T05:22:17.676+0000] {standard_task_runner.py:60} INFO - Started process 2115 to run task
[2025-06-06T05:22:17.678+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'LR_duration_prediction_training', 'train_model', 'manual__2025-06-06T05:22:15.889534+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/LR_duration-prediction_dag.py', '--cfg-path', '/tmp/tmp9rtwiaya']
[2025-06-06T05:22:17.679+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask train_model
[2025-06-06T05:22:17.690+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-06T05:22:17.714+0000] {task_command.py:423} INFO - Running <TaskInstance: LR_duration_prediction_training.train_model manual__2025-06-06T05:22:15.889534+00:00 [running]> on host a901f37d6140
[2025-06-06T05:22:17.775+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='LR_duration_prediction_training' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T05:22:15.889534+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-06T05:22:15.889534+00:00'
[2025-06-06T05:22:37.953+0000] {logging_mixin.py:188} INFO - Loaded records: 3403766
[2025-06-06T05:22:43.990+0000] {logging_mixin.py:188} INFO - Records after preparation: 3316216
[2025-06-06T05:23:03.397+0000] {logging_mixin.py:188} INFO - Loaded records: 3288250
[2025-06-06T05:23:08.705+0000] {logging_mixin.py:188} INFO - Records after preparation: 3199715
[2025-06-06T05:23:25.202+0000] {logging_mixin.py:188} WARNING - 2025/06/06 05:23:25 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-06-06T05:23:29.174+0000] {logging_mixin.py:188} INFO - Model intercept: 23.85
[2025-06-06T05:23:30.576+0000] {logging_mixin.py:188} WARNING - 2025/06/06 05:23:30 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-06-06T05:23:30.595+0000] {logging_mixin.py:188} WARNING - 2025/06/06 05:23:30 INFO mlflow.tracking._tracking_service.client: üèÉ View run auspicious-lamb-4 at: http://mlflow:5000/#/experiments/2/runs/b790abd1a1f148e0b12bddf9931ef45a.
[2025-06-06T05:23:30.596+0000] {logging_mixin.py:188} WARNING - 2025/06/06 05:23:30 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/2.
[2025-06-06T05:23:30.605+0000] {logging_mixin.py:188} INFO - Model trained and logged with run_id: b790abd1a1f148e0b12bddf9931ef45a
[2025-06-06T05:23:30.812+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-06-06T05:23:30.821+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=LR_duration_prediction_training, task_id=train_model, execution_date=20250606T052215, start_date=20250606T052217, end_date=20250606T052330
[2025-06-06T05:23:30.878+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-06T05:23:30.890+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
